lighty-modules/lighty-netconf-sb/src/main/java/io/lighty/modules/southbound/netconf/impl/util/NetconfConfigUtils.java

```diff
@@ -109,6 +109,62 @@ public static NetconfConfiguration createNetconfConfiguration(
         return netconfConfiguration;
     }
 
+    /**
+     * Load netconf southbound configuration from InputStream containing JSON data.
+     *
+     * @param jsonConfigInputStream InputStream containing Netconf config. data in JSON format.
+     * @param encryptionService custom AAAEncryptionService provided by user to be injected.
+     * @param clientDispatcher dispatcher from running lighty services.
+     * @return Object representation of configuration data.
+     * @throws ConfigurationException In case JSON configuration cannot be deserializable to JSON
+     *                                tree nodes or cannot bind JSON tree node to type.
+     */
+    public static NetconfConfiguration getNetconfConfiguration(final InputStream jsonConfigInputStream,
+                                                               final NetconfClientDispatcher clientDispatcher,
+                                                               final AAAEncryptionService encryptionService)
+            throws ConfigurationException {
+        final NetconfConfiguration configuration = createNetconfConfiguration(jsonConfigInputStream);
+        configuration.setClientDispatcher(clientDispatcher);
+        configuration.setAaaService(encryptionService);
+        return configuration;
+    }
+
+    /**
+     * Create default Netconf Southbound configuration, with custom dispatcher and encryptionService.
+     *
+     * @param configurationFrom other NetconfConfiguration to be created from.
+     * @param encryptionService custom AAAEncryptionService provided by user to be injected.
+     * @param clientDispatcher dispatcher from running lighty services.
+     * @return Object representation of configuration data.
+     */
+    public static NetconfConfiguration getNetconfConfiguration(final NetconfConfiguration configurationFrom,
+                                                               final NetconfClientDispatcher clientDispatcher,
+                                                               final AAAEncryptionService encryptionService) {
+        final NetconfConfiguration configuration = new NetconfConfiguration();
+        configuration.setClusterEnabled(configurationFrom.isClusterEnabled());
+        configuration.setTopologyId(configurationFrom.getTopologyId());
+        configuration.setWriteTxTimeout(configurationFrom.getWriteTxTimeout());
+        configurationFrom.setClientDispatcher(clientDispatcher);
+        configurationFrom.setAaaService(encryptionService);
+        return configuration;
+    }
+
+    /**
+     * Create default Netconf Southbound configuration from other NetconfConfiguration.
+     *
+     * @param configurationFrom other NetconfConfiguration to be created from.
+     * @return Object representation of configuration data.
+     */
+    public static NetconfConfiguration getNetconfConfiguration(final NetconfConfiguration configurationFrom) {
```

Comment:
I dont see the use of this method, is it suppose to copy config? I would delete this .. also lines 163 164 are wrong anyways.